<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>160</width>
    <height>174</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>160</width>
    <height>174</height>
   </size>
  </property>
  <property name="acceptDrops">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>GBmu</string>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>160</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionLoad"/>
    <addaction name="actionSave_state"/>
    <addaction name="actionLoad_state"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="actionDebug"/>
   </widget>
   <widget class="QMenu" name="menuControls">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="title">
     <string>Controls</string>
    </property>
    <addaction name="actionPlay"/>
    <addaction name="actionPause"/>
    <addaction name="actionStop"/>
    <addaction name="actionReset"/>
    <addaction name="actionMute"/>
   </widget>
   <widget class="QMenu" name="menuSpeed">
    <property name="title">
     <string>Speed</string>
    </property>
    <addaction name="actionSpeed_x1"/>
    <addaction name="actionSpeed_x2"/>
    <addaction name="actionSpeed_x4"/>
   </widget>
   <widget class="QMenu" name="menuModel">
    <property name="title">
     <string>Model</string>
    </property>
    <addaction name="actionModelAuto"/>
    <addaction name="actionModelDMG"/>
    <addaction name="actionModelCGB"/>
    <addaction name="separator"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuControls"/>
   <addaction name="menuTools"/>
   <addaction name="menuModel"/>
   <addaction name="menuSpeed"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="maximumSize">
    <size>
     <width>77777</width>
     <height>30</height>
    </size>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="movable">
    <bool>false</bool>
   </property>
   <property name="iconSize">
    <size>
     <width>18</width>
     <height>18</height>
    </size>
   </property>
   <property name="floatable">
    <bool>false</bool>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionLoad"/>
   <addaction name="actionDebug"/>
   <addaction name="actionPlay"/>
   <addaction name="actionPause"/>
   <addaction name="actionStop"/>
   <addaction name="actionMute"/>
  </widget>
  <action name="actionLoad">
   <property name="icon">
    <iconset resource="../resources/resources.qrc">
     <normaloff>:/load.png</normaloff>
     <normalon>:/loadROM.png</normalon>:/load.png</iconset>
   </property>
   <property name="text">
    <string>Load</string>
   </property>
   <property name="toolTip">
    <string>Load (+0)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionDebug">
   <property name="icon">
    <iconset resource="../resources/resources.qrc">
     <normaloff>:/debug.png</normaloff>:/debug.png</iconset>
   </property>
   <property name="text">
    <string>Debug</string>
   </property>
   <property name="toolTip">
    <string>Debug (+D)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+D</string>
   </property>
  </action>
  <action name="actionPlay">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../resources/resources.qrc">
     <normaloff>:/play.png</normaloff>:/play.png</iconset>
   </property>
   <property name="text">
    <string>Play</string>
   </property>
   <property name="toolTip">
    <string>Play</string>
   </property>
  </action>
  <action name="actionPause">
   <property name="icon">
    <iconset resource="../resources/resources.qrc">
     <normaloff>:/pause.png</normaloff>:/pause.png</iconset>
   </property>
   <property name="text">
    <string>Pause</string>
   </property>
  </action>
  <action name="actionStop">
   <property name="icon">
    <iconset resource="../resources/resources.qrc">
     <normaloff>:/stop.png</normaloff>:/stop.png</iconset>
   </property>
   <property name="text">
    <string>Stop</string>
   </property>
  </action>
  <action name="actionReset">
   <property name="icon">
    <iconset resource="../resources/resources.qrc">
     <normaloff>:/reset.png</normaloff>:/reset.png</iconset>
   </property>
   <property name="text">
    <string>Reset</string>
   </property>
   <property name="toolTip">
    <string>Reset (+R)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="actionSpeed_x1">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../resources/resources.qrc">
     <normaloff>:/speed.png</normaloff>:/speed.png</iconset>
   </property>
   <property name="text">
    <string>speed x1</string>
   </property>
   <property name="toolTip">
    <string>Speed x1</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+1</string>
   </property>
  </action>
  <action name="actionSpeed_x2">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../resources/resources.qrc">
     <normaloff>:/speed.png</normaloff>:/speed.png</iconset>
   </property>
   <property name="text">
    <string>speed x2</string>
   </property>
   <property name="toolTip">
    <string>Speed x2</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+2</string>
   </property>
  </action>
  <action name="actionSpeed_x4">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../resources/resources.qrc">
     <normaloff>:/speed.png</normaloff>:/speed.png</iconset>
   </property>
   <property name="text">
    <string>speed x4</string>
   </property>
   <property name="toolTip">
    <string>Speed x4</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+4</string>
   </property>
  </action>
  <action name="actionModelAuto">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Auto</string>
   </property>
  </action>
  <action name="actionModelDMG">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>DMG</string>
   </property>
  </action>
  <action name="actionModelCGB">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>CGB</string>
   </property>
  </action>
  <action name="actionMute">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="../resources/resources.qrc">
     <normaloff>:/soundOn.png</normaloff>
     <normalon>:/soundOff.png</normalon>:/soundOn.png</iconset>
   </property>
   <property name="text">
    <string>Mute</string>
   </property>
   <property name="toolTip">
    <string>Mute (+M)</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+M</string>
   </property>
  </action>
  <action name="actionSave_state">
   <property name="text">
    <string>Save state</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionLoad_state">
   <property name="text">
    <string>Load state</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Alt+S</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="../resources/resources.qrc"/>
 </resources>
 <connections/>
</ui>
